www/index.html

```diff
@@ -0,0 +1,124 @@
+<!doctype html>
+<html>
+<head>
+  <meta http-equiv=Content-Type content="text/html; charset=utf-8">
+  <title>NET calculator</title>
+  <link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet" href="css/net_opt_grid.css">
+</head>
+<body>
+
+  <h1>NET Calculator</h1>
+  WebGui and python library <a href="NET_details.html">documentation</a>. Version: 0.3 (<a href="version_history.html">history</a>). <a href="https://www.cfa.harvard.edu/~spaine/am/download/index.html">am</a> version: 9.2.<br>
+Bug/feature/suggestion requests: dbarkats [at] cfa.harvard.edu
+
+<form action="/cgi-bin/NETweb.py" method="post" enctype="multipart/form-data" id="NETopts" target="results">
```

Comment:
The form action is different than currently implemented on Odyssey (where the wrapper script `NETweb.cgi` is used instead), so this cannot be used as-is there.